src/components/Reports/AdditionalSalaryRequest/CompleteForm/CompleteForm.tsx

```diff
@@ -107,6 +113,17 @@ export const CompleteForm: React.FC = () => {
 
   return (
     <AdditionalSalaryRequestSection title={selectedSection.title}>
+      <AccountInfoCard
+        name={name}
+        accountNumber={accountNumber}
+        primaryAccountBalance={primaryAccountBalance}
+        remainingAllowableSalary={remainingAllowableSalary}
+      />
+      <Typography variant="body1" paragraph>
```

Comment:
optional: I've started using `<Trans t={t}>` for longer blocks of text like this and it's nice because it automatically wraps instead of being one long line!